client/src/components/share/use-share.tsx

```diff
@@ -0,0 +1,36 @@
+import i18next from 'i18next';
+
+interface ShareTemplateProps {
+  superBlock: string;
+  block: string;
+}
```

Comment:
ShareProps is the same as breadCrumbProps, can you link both, to reduce the repeated types.

```suggestion
export type ShareProps = BreadCrumbProps
```